apps/vscode/src/app/workspace-tree/workspace-tree-item.ts

```diff
@@ -27,7 +27,6 @@ export type WorkspaceRouteTitle =
   | 'Xi18n'
   | 'Dep-Graph'
   | 'Connect'
-  | 'Settings'
```

Comment:
Rather than removing this UI shortcut to settings, could we instead use this shortcut to open angular-console's vscode settings pane? I think in the short term it'll help teach our users that we have migrated to using the native settings apis